src/components/EventOverview/EventsPage.tsx

```diff
@@ -112,7 +111,6 @@ const EventsPage = () => {
   };
 
   const handleDeleteEvent = async (event: Event) => {
```

Comment:
not sure if this is really within the scope but is this `async` here pointless?

Reply:
bump this just in case @wongcmichelle

Reply:
Good catch, thanks :)